src/Component/RegistrationForm.js

```diff
@@ -0,0 +1,278 @@
+import React, { Component } from 'react'
+import '../Stylesheet/RegistrationForm.css'
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props)
+    this.isInputsValid = true
```

Comment:
Why this is not a state and normal value? Also tell me the difference between two of them

Reply:
This normal value 'isInputsValid' is used to check and update its value after every field validation. If we use state and set its value it will render every time. The state will render the view once its value is set using the 'setState' method.